go/src/koding/kites/config/config.go

```diff
@@ -104,6 +104,36 @@ func (u *URL) WithPath(paths ...string) *URL {
 	return &URL{URL: &ur}
 }
 
+// StringOrBool implements json.Unmarshaler, which can parse
+// either boolean type or converts string to bool.
+type StringOrBool bool
+
+var _ json.Unmarshaler = (*StringOrBool)(nil)
+
+func (b *StringOrBool) UnmarshalJSON(p []byte) error {
```

Comment:
If i understand it correctly you could just use the json string tag then
It would read a string into a bool and write a bool as string
And you would not need to parse it every time you use it.

```
type Konfig struct {
       ......
        Debug StringOrBool `json:"debug,string"`
        .....
}

```